app/routes/github.webhook/pull-request-webhook.ts

```diff
@@ -59,11 +59,13 @@ export async function processPullRequestWebhook(
       });
     }
     default:
+      // The payload type here is "never". We're casting it to any so we're able to log some properties for debug purposes
+      const unknownPayload = webhook.payload as any;
```

Comment:
Don't we define it here? https://github.com/iFixit/govinor/blob/6f08c7f568b03cf655aabc38522cb39a08fc575d/app/routes/github.webhook/pull-request-webhook.ts#L161-L165

Reply:
We've defined some payload type (I don't know if the list is exhaustive, is probably only what we need for the app as I didn't want to define zod schema for all the github interface)